exe/parlour

```diff
@@ -53,6 +73,13 @@ command :run do |c|
       break_params: configuration[:style][:break_params],
       tab_size: configuration[:style][:tab_size]
     )
+
+    Parlour::TypeLoader.load_project(
+      configuration[:root],
+      inclusions: configuration[:included_paths],
+      exclusions: configuration[:excluded_paths],
+      generator: gen,
+    )
```

Comment:
I'm happy for this to be the new default, but I think it would be a good idea to have an option to avoid loading the project, perhaps `--no-load` or something.

Some people may be using Parlour to generate "patch" RBI files with a small custom plugin, which add types to little bits of their project, but don't want a huge RBI generated for their entire project.

Reply:
I added the ability to disable parsing by moving the parsing-specific configuration into a single key. To disable, just set the key to `false`.

So you could customize the parsing behavior like:
```yaml
parser:
  included_paths:
    - lib
  excluded_paths:
    - spec
```

And disable it like:
```yaml
parser: false
```